### DISCUSSION TITLE
Next release

### DISCUSSION THREAD

**Discussion by nittka** [TVTower/TVTower#239] [General]
When do you plan to release the next version? Will it be a dev-version to get some more feedback before building? Is the build process documented somewhere?
...Don't forget to include the new docs folder.

**Comment by GWRon** [comment]
Before christmas?

Dev-Version at best ... until next week's end.

**Comment by nittka** [comment]
Congratulations for the 3000th commit.

**Comment by nittka** [comment]
I guess we are getting closer to the next release. Apart from the live time PR, I would like to fix the pool refill issue. Currently, there are no database definitions for that and there are only very few licence-flag definitions in the database, but they are not handled correctly for series definitions (return/sell to pool etc.). In order to start off consistently with the next Database/SaveGame version, I would prefer this issue to be addressed.
All other issues seem to fall into the improvements category that could wait for the following release.

Could we start using milestones? I would find them helpful for distinguishing tickets that serve as reminder (should be addressed at some point - backlog) and tickets that should be dealt with soon (assigned a milestone, e.g. the upcoming release).

**Comment by GWRon** [comment]
Of course we could add milestones...if it benefits the development then we could give it a whirl.

**Comment by nittka** [comment]
In case we want to release around Christmas again, I think we are running out of time. There are still some blockers and with the change in variable resolution wrt. different languages, doing quality control on the database (@scr0llbaer9000 ?)  will take some time as well...
